doc/releases/index.rst

```diff
@@ -72,6 +72,7 @@ Release timeline
 .. _18.2.1: reef#v18-2-1-reef
 .. _18.2.2: reef#v18-2-2-reef
 .. _18.2.4: reef#v18-2-4-reef
+.. _18.2.4: reef#v18-2-5-reef
```

Comment:
I think a copy/paste/edit omission -- 18.2.4 should be 18.2.5?

doc/releases/releases.yml should also be edited to match, taking care to maintain descending order in that file.